### PR TITLE
copr/mysql/json: support json_length

### PR DESCRIPTION
**Pull request by hawkingrei**
###  What have you changed?

Make JSON support the json_length. It's the implementation for JSON_LENGTH in Mysql.

###  What is the type of the changes?
- New feature

###  How is the PR tested?
- Unit test

###  Does this PR affect tidb-ansible?

- No.

### CODE REVIEW

File: components/tidb_query/src/codec/mysql/json/json_length.rs [hunk 1]
```diff
@@ -0,0 +1,97 @@
+// Copyright 2017 TiKV Project Authors. Licensed under Apache-2.0.
```
Comment (5kbpers) [inline components/tidb_query/src/codec/mysql/json/json_length.rs]: I think it should be 2019.

Comment (hawkingrei) [reply components/tidb_query/src/codec/mysql/json/json_length.rs]: done

**Pull request merged**